### PR TITLE
Replace travis with github actions

### PR DESCRIPTION
**Pull request by luca-rath**

### CODE REVIEW

**Comment by luca-rath** [comment]
See CI results here https://github.com/luca-rath/husky/pull/1

File: .github/workflows/test-application.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Setup Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: ${{ matrix.node-version }}
+
+            - uses: actions/cache@v2
```
Comment (nnatter) [inline .github/workflows/test-application.yaml:33]:
```suggestion
            - name: Cache dependencies
              uses: actions/cache@v2
```

File: .github/workflows/test-application.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                  restore-keys: |
+                      ${{ runner.os }}-node-
+
+            - name: Install dependencies
+              run: |
+                  npm install -g grunt-cli
+                  npm install
+
+            - name: Execute test cases
+              run: grunt ci --force
```
Comment (nnatter) [inline .github/workflows/test-application.yaml:46]: whats the reason for `--force`? 🙂

Comment (luca-rath) [reply .github/workflows/test-application.yaml:47]: I have no idea why this fails without the `--force` ... See https://github.com/luca-rath/husky/runs/1701422017

Comment (nnatter) [reply .github/workflows/test-application.yaml:47]: alright - looks like this failed on travis too: https://travis-ci.org/github/massiveart/husky

**Review by nnatter — APPROVED** [review]

**Comment by nnatter** [comment]
Looks good from my side. Unfortunately I am not allowed to merge anything here 😕

**Pull request merged**